### ISSUE TITLE
Unenrolled ChromeOS boot not working.

### ISSUE THREAD

**Issue by LavedenC1** [ading2210/shimboot#20]
**Describe the bug**
On the shimboot menu, if I try selecting chrome root a or b. Then it just shows the chrome os boot splash for a few minutes than darkness. No key or key combination helps.

**To Reproduce**
1. Go to the shimboot menu
2. Click on ChromeOS ROOT-A/B
3. Doesn't boot

**Expected behavior**
Supposed to boot

**Target Chrome OS Device (please complete the following information):**
 - Board Name Octopus
 - Device Name Fleex Dell Chromebook 3100
 - Shimboot version Prebuilt Image.

**Build Device (please complete the following information):**
 - OS: Chrome 123
 - Neofetch out [run `neofetch --stdout` and paste it here]

**Comment by ading2210** [comment]
Try recovering to an older Chrome OS version first. This might be because the shim kernel is too old to boot the newer Chrome OS release.

**Comment by LavedenC1** [comment]
It gives me an "Out of Date error" when I try to recover to an older image. Also my kernver is 0x00010003, so I can't downgrade.

**Comment by zek-c** [comment]
might as well wrap it up bro

**Comment by zek-c** [comment]
also have you tried waiting? Some cases require you to wait for up to 2 minutes for it to boot

**Comment by LavedenC1** [comment]
10 minutes of waiting.

**Comment by LavedenC1** [comment]
I built an image myself but it still doesn't work, I'm also using x86_64 hardware.

**Comment by LavedenC1** [comment]
I also got a new chromebook, with its kernel version lower. Same problem.

**Comment by ading2210** [comment]
This might be because your Chrome OS version is too new for the older shim kernel. Try recovering to an older version such as v105. I'm planning on overhauling this feature anyways so you can also probably just wait a few weeks.

**Comment by nathblade16** [comment]
> It gives me an "Out of Date error" when I try to recover to an older image. Also my kernver is 0x00010003, so I can't downgrade.

boot the recovery image like how you would shimboot

**Comment by kkilobyte** [comment]
> > It gives me an "Out of Date error" when I try to recover to an older image. Also my kernver is 0x00010003, so I can't downgrade.
> 
> boot the recovery image like how you would shimboot

that's how they get stuck on the logo because the shim kernel is too old, the octopus shim only gets Linux kernel 4.14

**Comment by HelloIloveGimkit2** [comment]
Yeah since I tested shimboot on a dedede board and it worked but when I tried my octopus board it didn't.

**Comment by lockieluke** [comment]
i'm on jacuzzi and it's on stuck on the chromeOS logo
